conda_forge_tick/migrators/arch.py

```diff
@@ -205,6 +213,12 @@ def __init__(
             not self.check_solvable
         ), "We don't want to check solvability for arm osx!"
 
+        # Excluded dependencies need to be removed before no target_packages are
+        # filtered out so that if a target_package is excluded, its dependencies
+        # are not added to the graph
+        for name in self.excluded_dependencies:
```

Comment:
name is used below, can you move line 228 up above this block?

Reply:
Thanks. Fixed